test/test_authn.py

```diff
@@ -51,7 +51,7 @@ def server(request, login_type):
 
     hw_repo = mxcubecore.HardwareRepository.get_hardware_repository()
     lims = hw_repo.get_hardware_object("lims")
-    lims.set_property("loginType", login_type)
+    lims._set_property("loginType", login_type)
 
```

Comment:
This does not look correct to me.

I did this change: https://github.com/elmjag/mxcubeweb/commit/14b5bbd35853fc6e09705570fe70ea755ad1abe3
With the change above, at least locally, all the `pytest` tests pass for me.